README.md

```diff
@@ -78,8 +82,14 @@ making a new tag during step 2. This is essentially redeploying an older version
         - To: The phone number to send the final confirmation text to (i.e. your phone number)
         - From: Any number in the format +12223334444 (we don't use this number for anything, but Twilio will complain if it's missing or incorrectly formatted)
 
+## 8. Celebrate
+
 1. send a message to the `#buttons-aa-general` Slack channel letting everyone know that the deployment is finished and list the changes in this deployment from the `CHANGELOG`
 
+1. update the ClickUp Tasks
+
+1. if appropriate, send a Feature Change Announcement email to the clients
```

Comment:
Out of curiosity - is there a template/example for this type of email that in the future we should refer to?

Reply:
I've sent some out using HubSpot to Sensors clients before. I don't remember the last one that I did for a Buttons change. But in either case, there is a template that Dana created that I used for these, but after the first one I just copied the previous and made modifications. I'll forward one of the previous ones to you all so you can see what they have looked like in the past.

Basically, if the change that we make is one that our clients should or might care about, then we like to show off and tell them about it. Dana will know how to do this and she might also post stuff on our social media about it.

Reply:
Thanks Theresa! This is really good go know going forward.